clinica/iotools/converters/adni_to_bids/adni_modalities/adni_t1.py

```diff
@@ -1,7 +1,7 @@
 """Module for converting T1 of ADNI."""
 from os import PathLike
 from typing import List, Optional
-
+import time
```

Comment:
Was it missing ?

Reply:
Time was not missing, I was previously using it for debugging and it is leftover, it can be removed